
2024-07-05 14:03:26,521 - root - logging_functions.py - ERROR - <module> - Testing saving logs to file.
